fe/fe-core/src/main/java/com/starrocks/catalog/IcebergTable.java

```diff
@@ -391,6 +392,11 @@ public boolean supportPreCollectMetadata() {
         return true;
     }
 
+    @Override
+    public boolean supportTimeTravel() {
```

Comment:
```suggestion
    public boolean isTemporal() {
```
How about using Temporal?

Reply:
I will change it in next patch